src/BenchmarksServer/Startup.cs

```diff
@@ -421,8 +422,18 @@ private static string CloneRestoreAndBuild(string path, ServerJob job, string do
             // Project versions must be higher than package versions to resolve those dependencies to project ones as expected.
             // Passing VersionSuffix to restore will have it append that to the version of restored projects, making them
             // higher than packages references by the same name.
-            ProcessUtil.Run(dotnetExecutable, "restore /p:VersionSuffix=zzzzz-99999", workingDirectory: benchmarksApp, environmentVariables: env);
-            ProcessUtil.Run(dotnetExecutable, $"build -c Release", workingDirectory: benchmarksApp, environmentVariables: env);
+            ProcessUtil.Run(dotnetExecutable, "restore /p:VersionSuffix=zzzzz-99999", throwOnError: false, workingDirectory: benchmarksApp, environmentVariables: env);
+
+            if (job.EnableRuntimeStore)
+            {
+                ProcessUtil.Run(dotnetExecutable, $"build -c Release", throwOnError: false, workingDirectory: benchmarksApp, environmentVariables: env);
+            }
+            else
+            {
+                // Publishing the application will prevent the runtime store from being used as it copies the assemblies in the published folder
+                ProcessUtil.Run(dotnetExecutable, $"publish -c Release -o {Path.Combine(benchmarksApp, "published")}", throwOnError: false, workingDirectory: benchmarksApp, environmentVariables: env);
+            }                
```

Comment:
@JunTaoLuo: Is there a more explicit way to enable/disable runtime store?  Relying on the different behavior of `dotnet build` and `dotnet publish` seems fragile.

Reply:
Not really. The binaries are loaded given a hierarchy. If the store is installed (beside the dotnet.exe or in the global installed location), it will be used. The only way to bypass it is to deploy app local which takes precedence over the installed runtime stores.

Reply:
OK, sounds like using `build` and `publish` is currently the best way to toggle between enabling and disabling the store.